### DISCUSSION TITLE
Removed access to repository

### DISCUSSION THREAD

**Discussion by ghost** [community/community#143333] [Repositories]
### Select Topic Area

Question

### GitHub Feature Area

Repositories

### Body

I linked the repository to my Whatsapp number. Then, after some time, I needed to run Facebook ads with that same number. So, I had to change the number to a WhatsApp business account. 

After that, the bot was deactivated. I deleted the codespace. Then, I created another codespace and I linked it. 

Then, this noon, I got an email that I've been declined access to the repository and it been removed. What should I do?

**Comment by ebndev** [comment]
Hey @Erhuvwu-tech!  👋 

Thanks for posting in the GitHub Community, USERNAME! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by Prince-919** [comment]
Response#2

**Comment by neha46** [comment]
Res#1

**Comment by neha46** [comment]
Response #2

**Reply by Prince-919** [reply]
dome
